### DISCUSSION TITLE
Align variables on project screen

### DISCUSSION THREAD

**Discussion by ShrubRustle** [ScratchAddons/ScratchAddons#3770] [Ideas]
A lot of interfaces people make in Scratch rely on visible variables being in a certain place on the stage, to align properly with the rest of the UI.

However, there's no good way to ensure the variable display is properly aligned/spaced, so they tend to feel a bit messy. I'd love a way to set a specific x/y coordinate for a variable - maybe in the 'variable manager' tab? Or, maybe, instead of that, there could be some sort of snap-to-grid system for moving them around on the stage?

**Comment by CST1229** [comment]
Maybe Shift+Drag to snap to horizontal/vertical dragging.
